### DISCUSSION TITLE
Material Theme Builder

### DISCUSSION THREAD

**Discussion by wkmhv** [unoplatform/uno#17319] [General]
in my UNO project I find the “Styles” folder and here “ColorPaletteOverride.xaml” and “ColorPaletteOverride.zip”.
How do I create “ColorPaletteOverride.zip” as described here?
https://material-foundation.github.io/material-theme-builder/

Although the Material Theme Builder Tool doesn't export Material Tokens (DSP) packages anymore, to import DSP packages just follow the same steps described previously and save the downloaded DSP zip file as ColorPaletteOverride.zip in the Styles folder of your application project. In that case, make sure to delete the material-theme.json file from Styles folder before building your application, to avoid conflicts.
????

**Comment by rajamatt** [comment]
Hi @wkmhv,

If your Uno app was created using an older version of the Uno app creation wizard/template, you will indeed find the ColorPaletteOverride.**zip** instead of the newer ColorPaletteOverride.**json** that the Material Theme Builder can output.

Sadly, although Uno still supports the zip file for backward compatibility, the Material Theme Builder Website no longer offers the possibility of exporting zip files.

So that we can further assist you, could you provide the version of Uno your app is using (4.8, 4.9, 4.10, 5.0, ...)? As well as the version of [Uno.Templates](https://www.nuget.org/packages/Uno.Templates) that you used to create the project, if you created it recently?

**Comment by wkmhv** [comment]
![image](https://github.com/unoplatform/uno/assets/50765701/b71250d4-3d0b-4e58-93f4-95ca9abc2f85)
![image](https://github.com/unoplatform/uno/assets/50765701/42ec72ae-e876-464a-9372-62ca1ac63622)
![image](https://github.com/unoplatform/uno/assets/50765701/7508ab4b-c1ca-41cc-9867-9fe1973a5d81)

After the update it now seems OK.

**Reply by rajamatt** [reply]
@wkmhv thanks for the follow up, glad to see it is working now.

Thank you for pointing out that the documentation on Material Theme Builder is confusing, we will be cleaning it up shortly and providing more info on upgrade scenarios like this one.
